lib/internal/util/inspect.js

```diff
@@ -2190,6 +2195,10 @@ function hasBuiltInToString(value) {
     builtInObjects.has(descriptor.value.name);
 }
 
+function returnFalse() {
+  return false;
+}
```

Comment:
is this faster than nulling out `hasOwnToPrimitive` and checking for that, avoiding the function call?

Reply:
That likely depends on a) the inlining and b) if it is more common to have to check for both methods (if that's the case, any additional check is more expensive).